style_utilities/variables.css

```diff
@@ -0,0 +1,47 @@
+:root {
+  /* Colors */
```

Comment:
Add a primary color variable

Reply:
Using a variable for primary color, not named it because of not having a fixed color yet.